build.sh

```diff
@@ -40,12 +40,19 @@ build_one() {
                 */NotoSansDevanagariUI-MM.plist)
                     g="${g/UI/}"
                     f='Noto Sans Devanagari UI'
-                    fontmake -g "$g" --mti-source "$1" --family-name "$f"
                     ;;
                 *)
-                    fontmake -g "$g" --mti-source "$1"
+                    f=''
                     ;;
             esac
+            if [[ -n "$f" ]]; then
```

Comment:
They have the same .glyphs source, but different family names designating certain instances to interpolate from that source, and different .plist files designating certain MTI feature sources to apply.

The reason I did this separate check outside of the case statement was in anticipation of multiple special cases like this where we have a .glyphs source containing multiple families.